jbpm-xes/src/main/java/org/jbpm/xes/XESExportMain.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.xes;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import org.apache.commons.cli.*;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.commons.io.FileUtils;
+import org.jbpm.xes.dataset.DataSetService;
+import org.jbpm.xes.dataset.DataSetServiceImpl;
+
+public class XESExportMain {
+
+    private static final String DS_NAME = "jdbc/xesDs";
+
+    public static void main(String[] args) {
+        // create the Options
+        Options options = getOptions();
+
+        System.setProperty(Context.INITIAL_CONTEXT_FACTORY,
```

Comment:
skip JNDI and supply data source directly

Reply:
@mswiderski yeah, good catch, I removed the JNDI completely now.